rust/crd/src/lib.rs

```diff
@@ -1124,18 +1081,10 @@ mod tests {
         let expected: BTreeMap<String, String> = vec![
             ("spark.executor.cores".to_string(), "2".to_string()),
             ("spark.executor.memory".to_string(), "128m".to_string()), // 128 and not 512 because memory overhead is subtracted
-            (
-                "spark.kubernetes.executor.limit.memory".to_string(),
-                "512m".to_string(),
-            ),
             (
                 "spark.kubernetes.executor.request.cores".to_string(),
                 "2".to_string(),
```

Comment:
Sorry for the dump question, but why do we request 2 cores when the users sets 250m?